themes/simple/css/admidio.css

```diff
@@ -609,3 +609,18 @@ tr.admidio-event-highlight {
 #plugin-login-form .btn {
     margin-bottom: 1rem;
 }
+
+
+/***********************************/
+/* Changelog configuration  */
+/***********************************/
+
+div.changelog-tableselect-header {
+    font-weight: bold;
+}
+ul.changelog-tableselect-list {
+    list-style-type: none;
+}
+ul.changelog-tableselect-list li label.changelog-tableselect-label {
```

Comment:
Instead of setting this explicitly in the global css file you can set the class `fw-normal` in the template at that place.

Reply:
Thanks, done.